src/EPPlus/Drawing/Vml/BaseSizing.cs

```diff
@@ -0,0 +1,385 @@
+            }
+            else
+            {
+                ExcelWorksheet ws = _ws;
+                decimal mdw = ws.Workbook.MaxFontWidth;
+
+                pix = 0;
+                for (int col = 0; col < From.Column; col++)
+                {
+                    pix += ws.GetColumnWidthPixels(col, mdw);
+                }
+                pix += From.ColumnOff / EMU_PER_PIXEL;
+            }
+
+            return pix;
+        }
+        internal int GetPixelTop()
+        {
+            int pix;
+            if (CellAnchor == eEditAs.Absolute)
+            {
+                pix = Position.Y / EMU_PER_PIXEL;
+            }
+            else
+            {
+                pix = 0;
+                var cache = _ws.RowHeightCache;
+                for (int row = 0; row < From.Row; row++)
+                {
+                    lock (cache)
+                    {
+                        if (!cache.ContainsKey(row))
+                        {
+                            cache.Add(row, _ws.GetRowHeight(row + 1));
+                        }
+                    }
+                    pix += (int)(cache[row] / 0.75);
+                }
+                pix += From.RowOff / EMU_PER_PIXEL;
+            }
+            return pix;
+        }
+
+        internal double GetPixelWidth()
+        {
+            double pix;
+            if (CellAnchor == eEditAs.TwoCell)
+            {
+                ExcelWorksheet ws = _ws;
+                decimal mdw = ws.Workbook.MaxFontWidth;
+
+                pix = -From.ColumnOff / (double)EMU_PER_PIXEL;
+                for (int col = From.Column + 1; col <= To.Column; col++)
+                {
+                    pix += (double)decimal.Truncate(((256 * ws.GetColumnWidth(col) + decimal.Truncate(128 / (decimal)mdw)) / 256) * mdw);
+                }
+
+                var w = (double)decimal.Truncate(((256 * ws.GetColumnWidth(To.Column + 1) + decimal.Truncate(128 / (decimal)mdw)) / 256) * mdw);
+                pix += Math.Min(w, Convert.ToDouble(To.ColumnOff) / EMU_PER_PIXEL);
+            }
+            else
+            {
+                pix = Size.Width / (double)EMU_PER_PIXEL;
+            }
+            return pix;
+        }
+        internal double GetPixelHeight()
+        {
+            double pix;
+            if (CellAnchor == eEditAs.TwoCell)
+            {
+                ExcelWorksheet ws = _ws;
+
+                pix = -(From.RowOff / (double)EMU_PER_PIXEL);
+                for (int row = From.Row + 1; row <= To.Row; row++)
+                {
+                    pix += ws.GetRowHeight(row) / 0.75;
+                }
+                var h = ws.GetRowHeight(To.Row + 1) / 0.75;
+                pix += Math.Min(h, Convert.ToDouble(To.RowOff) / EMU_PER_PIXEL);
+            }
+            else
+            {
+                pix = Size.Height / (double)EMU_PER_PIXEL;
+            }
+            return pix;
+        }
+
+        //internal void SetPixelTop(double pixels)
```

Comment:
Remove?

Reply:
Yeah, at least for now. Intended to use this as a potential base-class for both xmldrawings and vmldrawings (and maybe others) but couldn't get it to work. It would require a bit bigger of a re-factoring.